lighthouse-core/audits/deprecations.js

```diff
@@ -33,7 +33,7 @@ class Deprecations extends Audit {
     return {
       category: 'Deprecations',
       name: 'deprecations',
-      description: 'Site does not use deprecated APIs',
+      description: 'Avoids deprecated APIs',
```

Comment:
Ah, OK. Hm... Well, we could drop the verb... E.g. `Deprecated APIs`. Else it might require inversing the titles... E.g. `Uses Up-To-Date APIs`